framework/tests/all-systems/java8/memberref/Receivers.java

```diff
@@ -11,6 +11,7 @@
     R supply();
 }
 
+@SuppressWarnings("returnsreceiver:methodref.receiver.bound.invalid")
```

Comment:
Please include `// See Issue #2931` at the end of the line.  Also can you move the suppression just before the specific line causing the problem?  You should be able to write a `@SuppressWarnings` annotation just before a method declaration